packages/ripple-address-codec/src/index.test.ts

```diff
@@ -2,258 +2,272 @@ import {
   classicAddressToXAddress,
   xAddressToClassicAddress,
   isValidXAddress,
-  encodeXAddress
-} from './index'
+  encodeXAddress,
+} from "./index";
```

Comment:
i accidentally ran `eslint --fix` on this. i think we can just ignore these changes for now.